velox/connectors/hive/storage_adapters/s3fs/S3FileSystem.cpp

```diff
@@ -213,6 +213,21 @@ Aws::Utils::Logging::LogLevel inferS3LogLevel(std::string_view logLevel) {
   }
   return Aws::Utils::Logging::LogLevel::Fatal;
 }
+
+// Supported values are "Always", "RequestDependent", "Never"(default).
+Aws::Client::AWSAuthV4Signer::PayloadSigningPolicy inferPayloadSign(
+    std::string sign) {
+  // Convert to upper case.
+  std::transform(sign.begin(), sign.end(), sign.begin(), [](unsigned char c) {
```

Comment:
folly has an in-place `folly::toLowerAscii(str)` that is quite convenient to use

Reply:
Looks simple indeed. Will update to use this in a follow-up PR.